src/Take.Blip.Builder/Actions/ExecuteScriptV2/ScriptObjectConverter.cs

```diff
@@ -0,0 +1,148 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ClearScript;
+using Newtonsoft.Json;
+using Take.Blip.Builder.Actions.ExecuteScriptV2.Functions;
+
+namespace Take.Blip.Builder.Actions.ExecuteScriptV2
+{
+    /// <summary>
+    /// Utility converter to convert javascript result to c# representation on string.
+    /// </summary>
+    [SuppressMessage("ReSharper", "SuggestBaseTypeForParameter")]
```

Comment:
Could you create a StringExtension ([Extension Methods](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods)) in this case?

Reply:
I can, but don't know if it would be better or worse HAHA what do you think?

I think it doesn't make such a difference, since it is only used in our script action and should not be used in other parts of our codebase...